src/app/app.module.ts

```diff
@@ -78,6 +77,7 @@ import { PublishedPopupComponent } from './players/published-popup/published-pop
     MatListModule,
     MatPaginatorModule
   ],
+  schemas: [CUSTOM_ELEMENTS_SCHEMA],
```

Comment:
WHy the custom element scema is required ?

Reply:
Angular by default will not recognise  our custom <sunbird-pdf-player></sunbird-pdf-player> HTML tag, to help Angular know our Web Component we need to add a schemas property to the @NgModule decorator and add CUSTOM_ELEMENTS_SCHEMA inside the schemas array.